include/zmq.h

```diff
@@ -381,6 +381,7 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg_,
 #define ZMQ_GSSAPI_PRINCIPAL_NAMETYPE 90
 #define ZMQ_GSSAPI_SERVICE_PRINCIPAL_NAMETYPE 91
 #define ZMQ_BINDTODEVICE 92
+#define ZMQ_ONLY_FIRST_SUBSCRIBE 93
```

Comment:
93 already exists - needs to be added in the ZMQ_BUILD_DRAFT_API below, and duplicated in src/zmq_drafts.hpp

Reply:
Should be fixed now.

Reply:
It's still missing from src/zmq_drafts.hpp though? With that, there's no need to ifdef in the implementation, but only for external usage